test/python/sdpa/frost/test_sdpa_fwd_fp8_sm100.py

```
@@ -214,6 +231,50 @@ def test_fp8_output_dtypes(in_key, out_key):
     _check(O, O_ref, _OUT[out_key], in_key, a_s, a_s_ref, a_o, a_o_ref)
 
 
+@pytest.mark.L0
+@pytest.mark.parametrize("out_key", ["fp16", "bf16", "e4m3", "e5m2"])
+@pytest.mark.parametrize("in_key", ["e4m3", "e5m2"])
+@torch_fork_set_rng(seed=0)
+def test_fp8_d192_d128_output_dtypes(in_key, out_key):
+    """Exact DSv3 shape: FP8 Q/K use d192 while V and O use d128."""
+    scale = 1.0 / math.sqrt(192)
+    O, O_ref, a_s, a_s_ref, a_o, a_o_ref = _run(
```

Ruff (0.16.1)

[error] Ambiguous variable name: `O`

(E741)